spec/spec_helper.rb

```diff
@@ -1,3 +1,5 @@
+require 'database_cleaner'
+require 'capybara/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.